src/mono/mono/btls/btls-error.c

```diff
@@ -58,8 +58,8 @@ mono_btls_error_get_reason (int error)
         case SSL_R_NO_RENEGOTIATION:
             return 100;
         default:
-            return 0;
+            break;
     }
 
-    return reason;
+    return 0;
```

Comment:
Well okay - looks like I didn't see this and only looked at the old code.  But why not just delete this line and leave the other one as is?